### DISCUSSION TITLE
Options To Turn Off Raw Mouse Input In Lazer

### DISCUSSION THREAD

**Discussion by RaoraRare** [ppy/osu#25361] [Q&A] [answered]
I was wondering if the option to turn off raw mouse input was going to get add 
for the people who don't play with on it 

<img width="351" alt="screenshot001" src="https://github.com/ppy/osu/assets/120297366/2ef68d38-8be0-4a7a-8287-5ca0e4995a5b">

![osu_2023-11-04_16-45-36](https://github.com/ppy/osu/assets/120297366/2f5394c5-e1af-4fb1-bc10-ff6247a58d23)

**Comment by bdach** [comment, marked as answer]
It's the 'high precision mouse" toggle.

If you search for "raw" in the settings it's the only toggle that appears.

**Reply by RaoraRare** [reply]
Thank you
